torch/_dynamo/trace_rules.py

```diff
@@ -351,6 +352,9 @@
     "torch.sparse_csc_tensor": SkipFunctionVariable,
     "torch.sparse_csr_tensor": SkipFunctionVariable,
     "torch.sparse_compressed_tensor": SkipFunctionVariable,
+    # Specially handle system-level communication functions 
+    "torch.distributed.distributed_c10d.send": CommunicationFunctionVariable,
+    "torch.distributed.distributed_c10d.recv": CommunicationFunctionVariable,
```

Comment:
Help me understand why these aren't handled the same way as other traceable collectives?